src/app/admin/_components/Event/TagSelect.js

```diff
@@ -1,25 +1,49 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import CreatableSelect from 'react-select/creatable';
 import { Loading, useGetList, useCreate } from 'react-admin';
 import PropTypes from 'prop-types';
 import { RESOURCES } from '@admin/_lib/consts';
 
 export function TagSelect({ setSelectedTags, defaultValue }) {
+  let newTagsLength;
   const [create] = useCreate();
+  const [value, setValue] = useState(defaultValue);
+  const [error, setError] = useState(false);
   const { data: tags, isLoading } = useGetList(RESOURCES.eventTag);
+
   if (isLoading) return <Loading />;
+
+  const onChage = newTags => {
```

Comment:
Probably the name should be "onChange" or even "onTagChange" for better understanding.

Reply:
Typo, I'll change it

Reply:
changed it